forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -383,3 +383,35 @@ def post():
         except BusinessException as err:
             current_app.logger.warning(err.error)
             return err.error, err.status_code
+
+
+@cors_preflight("PUT,OPTIONS")
+@API.route("/form-update/<string:form_id>", methods=["PUT", "OPTIONS"])
```

Comment:
PUT is an update, no need for update in the path

Reply:
updated path